packages/metablocks/src/api.js

```diff
@@ -25,8 +39,257 @@ const createUniverse = async ({
   } catch (e) {
     console.log("from api", e);
     throw e;
+  } finally {
```

Comment:
Redundant `finally`. Also the `console.log` is a leftover from my dev flow. The `api` should just throw the error and the caller should take care of the errors.

This applies to a lot of places in this PR.

Reply:
done